File: Test/Docker-Testing/sample.py
def find_primes_up_to_n(n):
    primes = []
    for possiblePrime in range(2, n + 1):
        isPrime = True
        for num in range(2, int(possiblePrime ** 0.5) + 1):
            if possiblePrime % num == 0:
                isPrime = False
                break
        if isPrime:
            primes.append(possiblePrime)
    return primes

def main():
    primes_up_to_1000 = find_primes_up_to_n(1000)
    for prime in primes_up_to_1000:
        print(prime)

if __name__ == "__main__":
    main()

File: Test/Docker-Testing/sample.go
package main
import "fmt"

func main() {
    printPrimes()
}

func printPrimes() {
    for num := 2; num <= 10; num++ {
        if isPrime(num) {
            fmt.Println(num)
        }
    }
}

func isPrime(n int) bool {
    for i := 2; i*i <= n; i++ {
        if n%i == 0 {
            return false
        }
    }
    return true
}